src/plugins/score-plugin-deviceexplorer/Explorer/Explorer/Widgets/DeviceEditDialog.hpp

```diff
@@ -82,5 +83,6 @@ class SCORE_PLUGIN_DEVICEEXPLORER_EXPORT DeviceEditDialog final : public QDialog
 
   QString m_originalName{};
   int m_index{};
+  QSplitter* splitter;
```

Comment:
In C++ when we create some pointer member / pointer variable, it's very important to directly initialize it to nullptr, otherwise it will point to garbage data: 

    QSplitter* splitter{};